packages/cyberstorm/src/components/Dialog/Dialog.tsx

```diff
@@ -111,25 +79,8 @@ export function Dialog(props: DialogProps) {
                   noPadding ? null : styles.bodyPadding
                 )}
               >
-                {content}
+                {children}
               </div>
-
-              {hideFooter ? null : (
```

Comment:
I'm not sure if this is intentional, but as a headsup, if a footer is now passed as a child, it will be rendered inside the `div` above, meaning e.g. `styles.bodyPadding` will affect it as well.

Reply:
I'm aware. I'll be going over the Dialogs again on the second QA round and fixing any problems, that might have arisen.